cloudferrylib/os/storage/cinder_database.py

```diff
@@ -22,23 +23,106 @@
 
 CINDER_VOLUME = "cinder-volume"
 LOG = utl.get_log(__name__)
+ID = 'id'
+DISPLAY_NAME = 'display_name'
 PROJECT_ID = 'project_id'
 STATUS = 'status'
 TENANT_ID = 'tenant_id'
 USER_ID = 'user_id'
 DELETED = 'deleted'
 HOST = 'host'
-IGNORED_TBL_LIST = ('quota_usages')
+IGNORED_TBL_LIST = ('quotas', 'quota_usages')
 QUOTA_TABLES = (
     'quotas',
     'quota_classes',
     'quota_usages',
 )
 
+# SKIPPING_STATUSES = ['creating', 'error', 'error_deleting', 'attaching',
+# 'error_attaching', 'detaching']
+VALID_STATUSES = ['available', 'in-use']
```

Comment:
We should also keep cinder volumes which are in `attaching` and `detaching` state:
 - force status change to either "available" or "in-use"
 - if changed to "available" or "in-use" - add this volume to migration procedure
 - if unable to change to "available" or "in-use" - display warning message and proceed with migration